app/views/events/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="container posts-new">
+  <div class="row">
+
+  </div>
+</div>
+
+<div class="container emp-profile">
+  <div class="row">
+    <div class="col-lg-8 bg-e">
+
+    </div>
+    <div class="col-lg-4">
+      <h3>created by: <%= @event.user.username %></h3>
+      <h1><%= @event.name %></h1>
+      <h3><%= @event.event_date %></h3>
+      <h3><%= @event.location %></h3>
+      <h4><%= @event.description %></h4>
+
+      <% if @event.upcoming? %>
```

Comment:
- [ ] There shoud be absolutely NO logic in views.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png)

Please move the logic to helpers.